docs/ReleaseNotes.md

```diff
@@ -18,6 +18,22 @@ The included licenses apply to the following files:
 
 ## Changelog
 
+### Version 1.8.2407
+
+This cumulative release contains numerous bug fixes and stability improvments.
+
```

Comment:
Why the double-spacing of the bullet list items?

Reply:
Updated.

Reply:
I don't know why, but they still look double-spaced.  It looks like you replaced `*` with `-`, but didn't delete the extra newlines in-between them.

Reply:
Fixed.